tests/validate-schema.py

```diff
@@ -0,0 +1,54 @@
+#!/usr/bin/env python3
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation; either version 2 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+#
+# See LICENSE for more details.
+#
+# Copyright: Red Hat Inc. 2022
+# Author: Beraldo Leal <bleal@redhat.com>
+"""
+Validates autil YAML files against our schema.
+
+For now, JSON Schemas are being used to validate YAML documents.
+"""
+
+import glob
+import sys
+from pathlib import Path
+
+import yaml
+from jsonschema import exceptions, validate
+
+
+def validate_yaml(filename, schema):
+    """Validate a yaml file against a specific schema."""
+    print(f"Validating {filename}...")
+    with open(filename, 'r') as fp_data, open(schema, 'r') as fp_schema:
+        data = fp_data.read()
+        schema = fp_schema.read()
+
+        validate(yaml.load(data, Loader=yaml.SafeLoader),
+                 yaml.load(schema, Loader=yaml.SafeLoader))
+
+
+print("Starting validation...")
```

Comment:
Any reason why you're not wrapping this on an `if __name__ == "__main__":` block?

Reply:
There is a good one: I forgot! :) Thanks for noticing that. Just fixed with a force-push.